MdeModulePkg/Core/Dxe/Misc/MemoryProtection.c

```diff
@@ -932,86 +1234,244 @@ CoreInitializeMemoryProtection (
   VOID
   )
 {
+  // MU_CHANGE Start: Consolidate MU_CHANGEs

```

Comment:
We can delete some of this, but to keep with our current style, it is just commenting out, not deleting. This will be cleaned up with edk2 upstreaming.